examples/apply-setters-simple/.expected/diff.patch

```diff
@@ -26,4 +26,4 @@ index b521905..0d10662 100644
  environments: # kpt-set: ${env}
 +  - prod
    - dev
--  - stage
```

Comment:
We can avoid this new line diffs.

Reply:
We should have newline at the end of a file. And our auto update tool also makes newline at the end.
If you take a closer look, the real diff has not changed. This will only show up once.